contracts/contracts/RLPWriter.sol

```diff
@@ -0,0 +1,249 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @custom:attribution https://github.com/bakaoh/solidity-rlp-encode
```

Comment:
I would recommend importing directly from the git commit. If I'm auditing the code, I don't really want to spend time figuring out what changes you made. If I'm maintaining the code, I'd prefer for it not to exist; and if it does, I'd also want the original tests.

Reply:
I'm not sure how to cherry-pick an individual file from a range of commits from another repo.

Reply:
I'm going to leave this as-is for now, as the code is very straightforward.

Reply:
https://remix-ide.readthedocs.io/en/latest/import.html#import-from-a-github-url